src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationService.kt

```diff
@@ -228,7 +228,7 @@ class AdjudicationService(
     offenderNo: String,
     request: CreateAdjudicationRequest,
   ): AdjudicationResponse {
-    val adjudicationNumber = checkAdjudicationDoesNotExist(request.adjudicationNumber)
+    val adjudicationNumber = adjudicationIncidentPartyRepository.getNextAdjudicationNumber()
     val prisoner = findPrisoner(offenderNo)
```

Comment:
we can't use a generated JPA annotation since column is also null and is not primary key